common/text/tree_utils.h

```diff
@@ -106,6 +106,14 @@ const SyntaxTreeLeaf& GetSubtreeAsLeaf(const S& symbol,
       GetSubtreeAsSymbol(symbol, parent_must_be_node_enum, child_position)));
 }
 
+template <class S, class E>
+const E GetSubtreeNodeEnum(const S& symbol,
```

Comment:
by-value return types should never have `const`, so just `E GetSubtreeNodeEnum`

Reply:
Done